hypertrace-core-view-generator/build.gradle.kts

```diff
@@ -18,7 +18,7 @@ dependencies {
   implementation(project(":hypertrace-core-view-generator-api"))
   implementation("org.hypertrace.core.viewgenerator:view-generator-framework:0.1.16")
   implementation("org.hypertrace.core.datamodel:data-model:0.1.9")
-  implementation("org.hypertrace.core.spannormalizer:raw-span-constants:0.1.22")
+  implementation("org.hypertrace.traceenricher:enriched-span-constants:0.3.4")
```

Comment:
Hmm, we are using some none `core` library here, this may affect licensing stuff. Revert this and use Otel keys directly by defining them there now.

Reply:
Ok, I was not familiar with the idea behind `core` library. 
Let me update this.